sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/Infrastructure/StorageTestEnvironment.cs

```diff
@@ -28,7 +28,12 @@ public class StorageTestEnvironment: TestEnvironment
         public static StorageTestEnvironment Instance => Singleton.Value;
 
         /// <summary>The environment variable value for the storage account connection string, lazily evaluated.</summary>
-        private string StorageAccountConnectionString => GetOptionalVariable("EVENT_PROCESSOR_STORAGE_CONNECTION_STRING");
+        private string StorageAccountConnectionString => GetOptionalVariable("EVENTHUB_PROCESSOR_STORAGE_CONNECTION_STRING");
+
+        /// <summary>
+        ///   The storage account endpoint suffix of the cloud to use during Live tests.
+        /// </summary>
+        public new string StorageEndpointSuffix => base.GetOptionalVariable("STORAGE_ENDPOINT_SUFFIX") ?? "core.windows.net";
```

Comment:
nit: is `base.` required? I don't think you override this in the StorageTestEnvironment class.

Reply:
We do, yes.   We're intending to shadow a base class member.  Intending being the key here.   I, uh, forgot to update this to actually shadow that member after I committed the change to Azure.Core.   Fixed!